sound/soc/sof/intel/hda-dai.c

```diff
@@ -475,7 +473,7 @@ static int ipc4_hda_dai_trigger(struct snd_pcm_substream *substream,
 
 		pipeline->state = SOF_IPC4_PIPE_RESET;
 
-		ret = hda_link_dma_cleanup(substream, hext_stream, cpu_dai, codec_dai);
+		ret = hda_link_dma_cleanup(substream, hext_stream, dai, codec_dai);
```

Comment:
why not call it cpu_dai in the parameter list then? 'dai' adds ambiguity on why there's no prefix and there's one for 'codec_dai'.

Reply:
but why would I change the signature of the function unnecessarily? If you ook at the DAI drv ops, it is called dai and its implicit that it is the CPU DAI already

Reply:
I really don't see how going from 'cpu_dai, codec_dai' to 'dai, codec_dai' helps make the code more readable. It just confusing for the reviewer to see differences like this.